module/library/design/README.md

```
@@ -0,0 +1,43 @@
+## Design module
+
+Basic design system for application provides UI theme and components.
+
+### Colors Pallet
+
+Colors pallet is presented in M3 convention in presented contrast, eg:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)